### PR TITLE
[Triton/Gluon] Take the autotuning search out of the unit tests

### PR DESCRIPTION
**Pull request by Boss2002n**
The CI gate for runtime autotuning fired on three tests -- test_moe_wgrad, test_flash_kda and test_fast_launch. 

Every @triton.autotune now takes its config list from autotune_config

op_tests/triton_tests/__init__.py already switched one family off for pytest; it now pins one config for all of them, by rebinding prune_configs on the Autotuner class this process imported. 

The rule is written down in .github/instructions/aiter-ops-triton.instructions.md and in aiter/ops/triton/README.md.

Worth stating: searching several configs gave the suite incidental coverage of more than one schedule. That coverage was accidental and nondeterministic -- which config ran depended on load.

### CODE REVIEW

**Review by brunomazzottiamd — DISMISSED** [review]
LGTM!

**Comment by brunomazzottiamd** [comment]
Added `ci:triton-300x` label to trigger `gfx942` runners. I want to see how this autotuning stuff is behaving in `gfx942`.

Related to https://github.com/ROCm/aiter/pull/5594.

File: op_tests/triton_tests/chunk_delta_attn/test_flash_kda.py [hunk 1]
```diff
@@ -363,22 +383,22 @@ def test_tuner_keeps_the_two_schedules_apart():
...
-    kern = _flash_kda._flash_kda_segment_kernel
     args = make_inputs(1, 1024, 4)
     # An incoming state is what makes the two output passes otherwise identical:
     # without it the unsegmented one passes h_in=None and the key picks up the
     # difference through the dtypes it appends, hiding the collision.
     kw = {"initial_state": torch.zeros(1, 4, K_DIM, K_DIM, device=device)}
     # This is about the Triton kernel's autotuner, which never runs -- and whose
     # cache therefore stays empty -- when K2 is routed to Gluon.
-    with _route(k2=False):
+    with _k2_tuner_search_space() as kern, _route(k2=False):
```
[the remark this answers is not in the document]
Comment (Boss2002n) [reply op_tests/triton_tests/chunk_delta_attn/test_flash_kda.py:396]: The empty cache came from autotune_configs collapsing configs to one entry, which is what this context manager undoes.

**Review by brunomazzottiamd — DISMISSED** [review]

**Review by brunomazzottiamd — APPROVED** [review]
`op_tests/triton_tests/chunk_delta_attn/test_flash_kda.py` is fixed on `gfx942`, so far so good.

I'll merge this, but Flash Attention Integration is broken - all jobs failing in a fundamental way, unable to perform some sort of installation...

- Flash Attention Integration / Flash Attention - Triton / MI35X (1 GPU)
- Flash Attention Integration / Flash Attention - Triton / RDNA3 (1 GPU)
- Flash Attention Integration / Windows Smoke Test

@Boss2002n, can you please feed the logs to an AI agent and brainstorm a solution?

**Review by vgokhale — APPROVED** [review]
